Update staking page based on OpenTezos page on delegating (#512)
trilitech/tezos-developer-docs: docs/architecture/bakers.md

A delegate also participates in [governance](/architecture/governance) in proportion to their _voting power_.
The voting power of a delegate is computed in a similar way to the baking power except that no distinction is made between tez that are staked or not staked.

Bakers must run at least one Tezos node and a baker service to go with it, see our [tutorial for bakers](/tutorials/join-dal-baker).
These services must run at all times with a stable power source and internet connection, because periods of inactivity for a baker entail losses of rewards, and eventually being marked as inactive and temporarily excluded from baking.

> ## Delegating to a baker

Author: I think it's covered by step 1 in the staking section. We wanted to reduce the prominence of delegation here.

> ## Delegating to a baker

Reviewer: I understand, but on this page it's odd to have the comparison table at the end while not mentioning delegation at all. You see what I mean?

A delegate also participates in [governance](/architecture/governance) in proportion to their _voting power_.
The voting power of a delegate is computed in a similar way to the baking power except that no distinction is made between tez that are staked or not staked.

Bakers must run at least one Tezos node and a baker daemon to go with it.
These daemons must run at all times with a stable power source and internet connection, because periods of inactivity for a baker entail losses of rewards, and eventually being marked as inactive and temporarily excluded from baking.

## Delegating to a baker

> In older versions of the Tezos protocol, before non-baker users could stake, delegating was the primary way that users earned rewards from tez.

Reviewer: why "from tez"? Is it more like:
```suggestion
In older versions of the Tezos protocol, before users could stake with a baker, delegating was the primary way that users earned rewards for their tez.
```

Users delegated their accounts to a baker and in return the baker could choose to share some of their baking rewards with their delegators.

> The Tezos system did not automatically provide rewards to delegators.

Reviewer: ```suggestion
The rewards to delegators were distributed by the bakers, with no guarantee from the Tezos protocol.
```

> Delegating still works in this way, but currently, the primary use of delegating is as the first step in staking.
> To stake with a baker, users must first choose the baker as their delegate.

Reviewer: remove some redundancy?
```suggestion
Delegating still works in this way, but currently, it's mostly seen as a preliminary step required for staking.
```

> Then the system provides the staker with rewards automatically.

Reviewer: ```suggestion
In return, the Tezos protocol provides stakers with rewards automatically.
```
